src/jit/lower.cpp

```diff
@@ -648,6 +639,9 @@ void Lowering::LowerSwitch(GenTree* node)
     // Get rid of the GT_SWITCH(temp).
     switchBBRange.Remove(node->gtOp.gtOp1);
     switchBBRange.Remove(node);
+
+    // TODO(pdg): is this correct?
+    return switchBBRange.EndExclusive();
 }
```

Comment:
What is it supposed to return? (it would be good if there was a header comment that said this). Is it the last node we've lowered? In that case, this isn't correct. Computing it requires writing code for each case, above.